packages/manager/modules/pci/src/projects/project/kubernetes/details/service/update/update.routing.js

```diff
@@ -11,6 +14,12 @@ export default /* @ngInject */ ($stateProvider) => {
       resolve: {
         goBack: /* @ngInject */ goToKubernetesDetails => goToKubernetesDetails,
         breadcrumb: () => null,
+        isMinorVersionUpgrade: /* @ngInject */
+          $transition$ => $transition$.params().isMinorVersionUpgrade,
+        nextMinorVersion: /* @ngInject */ (cluster) => {
```

Comment:
Prefer using `nextVersion` (because you are return both `majorVersion` and `minorVersion`).

Reply:
Could be addressed in a separate Pull Request since this ticket has been validated.